rules/Multifactor-Google-Authenticator-Do-Not-Rename.js

```diff
@@ -16,19 +17,29 @@ function (user, context, callback) {
         return callback(null, user, context);
     }
 
-    getUserProfile(user)
-        .then(function (user_profile) {
-            if (!loggedInRecently(user_profile)) {
-                enableMFA(context);
-            }
-
-            return callback(null, user, context);
-        })
-        .catch(function(err) {
-            enableMFA(context);
-            return callback(null, user, context);
-        });
-
+    // Skip MFA if user's IP is on a corporate network
+    var CORPORATE_NETWORK_IPS = [
+        // https://github.com/ministryofjustice/moj-ip-addresses/blob/master/moj-cidr-addresses.yml
+        // DOM1 = disc_internet_pipeline
+        '157.203.176.138', '157.203.176.139',  // 157.203.176.138/31
+        '157.203.176.140',                     // 157.203.176.140/32
+        '157.203.177.190', '157.203.177.191',  // 157.203.177.190/31
+        '157.203.177.192',                     // 157.203.177.192/32
+        // MoJDigital Wifi
```

Comment:
FYI FWIW
81.134.202.29/32 is MOJDigital WiFi (in 102PF) and MOJ Digital's VPN (vpn.cjsdigital.org)

Reply:
Useful to know. I've added it.